.github/workflows/tests.yml

```diff
@@ -14,9 +14,9 @@ on:
     - '*'
 
 env:
-  CACHE_NUMBER: 0  # increase to reset cache manually
-  PYTEST_FLAGS: --tardis-refdata=${{ github.workspace }}/tardis-refdata
-                --cov=tardis --cov-report=xml --cov-report=html
+  CACHE_NUMBER: 1  # increase to reset cache manually
```

Comment:
Is this needed to merge?

Reply:
No, but it uses the old cache from the master branch otherwise. I would change this once everything else is done and this PR is approved.